modules/custom/entity_access_by_field/entity_access_by_field.post_update.php

```diff
@@ -8,6 +8,6 @@
 /**
  * Rebuild node access.
  */
-function entity_access_by_field_post_update_10101_rebuild_node_access() {
+function entity_access_by_field_post_update_10102_rebuild_node_access() {
```

Comment:
@Ressinel Please don't rename the existing one, instead, simply, add the comment link to this PR in the preview hook and create a new hook for rebuilding access.

Reply:
Check it, please.